src/core/CMakeLists.txt

```diff
@@ -702,7 +702,11 @@ SET(QGIS_CORE_SRCS
 
 FILE(GLOB JSON_HELP_FILES "${CMAKE_SOURCE_DIR}/resources/function_help/json/*")
 IF(NOT USING_NINJA)
-  STRING(REPLACE "$" "$$" JSON_HELP_FILES "${JSON_HELP_FILES}")
+  IF (APPLE)
+    STRING(REPLACE "$" "\$" JSON_HELP_FILES "${JSON_HELP_FILES}")
```

Comment:
So this was my fault? We did some changes to JSON files, was that the problem? Is there something I could have done to avoid it?

Reply:
I do not think it is your problem. I do not know something changed probably with CMake or Make or whoknows...